include/mbgl/style/layers/layer.hpp.ejs

```diff
@@ -44,6 +44,13 @@ public:
 <% } -%>
 
 <% } -%>
+    // Visibility
+    void setVisibility(VisibilityType) final;
```

Comment:
I'm specifically interested in `override` (`final override` reads better), to make it fully clear that this is a virtual function override.